### PR TITLE
Set up alembic for database migrations

### PR DESCRIPTION
**Pull request by tsauerwein**
Related to https://github.com/c2corg/v6_api/issues/476

### CODE REVIEW

File: alembic_migration/README.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+[documentation](http://alembic.zzzcomputing.com/en/latest/cookbook.html#replaceable-objects)
+is used.
+
+## Run a migration
+
+A migration should be run each time the application code is updated or if you have just created a migration script.
+
+```
+.build/venv/bin/alembic upgrade head
+```
```
Comment (asaunier) [inline alembic_migration/README.md]: This command should be run on the target server, right?

**Comment by asaunier** [comment]
All the triggers and function definitions are moved to an initial alembic script. Right?
When creating a new instance with a new DB, does this mean alembic is called automatically? I guess it's here https://github.com/c2corg/v6_api/blob/494d48dcc4097aba440d658994bef3e65eaa49b1/c2corg_api/scripts/initializedb.py#L42 right?

Just a question to understand how this works.

**Comment by asaunier** [comment]
@tsauerwein is there something still required before merging the PR? Except approval :P 

Looks good to me. I have tested the branch in my API instance. At least it still works :)

**Comment by tsauerwein** [comment]
I tested the migration this morning, and have to commit 1-2 small changes.

**Comment by tsauerwein** [comment]
@arnaud-morvan As a reminder: Once this is merged, you will have to adapt the script `migrate_topoguide` to also create the schema `alembic`.

File: alembic_migration/README.md (hunk 1 above)
Comment (tsauerwein) [inline alembic_migration/README.md]: Yes, on the database that should be migrated.

**Pull request merged**